### ISSUE TITLE
BandagedBD Not Installing

### ISSUE THREAD

**Issue by NotNorp** [rauenzi/BetterDiscordApp#96]
None of the installers work, they keep getting stuck on "Killing Discord Processes" 
What it looks like:
![bd issues](https://user-images.githubusercontent.com/44438187/47435171-4fe7cd80-d7a4-11e8-9ba7-06e3a87181d6.JPG)

Not sure what to do, my discord is already closed, its not open, not in task manager either. BetterDiscord's discord server is down so I have to post this here I guess.

**Comment by rauenzi** [comment]
Run the installer as admin

**Comment by Escaflowdra** [comment]
I just tried to run as admin since I'm having the same issue and it's still stuck on the same issue.

**Comment by Escaflowdra** [comment]
Update! Just killed discord from task manager, tried again and it worked when running as admin for me. Had to start up discord after it said it installed and it worked!

**Comment by NotNorp** [comment]
If anyone has this issue, just restart your pc. Discord had a secret process running that was NOT in my task manager. I've tried installing as admin, out of downloads folder, literally anything. BD staff told me to just reinstall discord entirely. Thanks for the help but I already did found out how to fix it like an hour ago. My only issue with BetterDiscord now is a lot of lagging but that's not this issue so not gonna start here.
